base_layer/wallet_ffi/wallet.h

```diff
@@ -461,6 +461,9 @@ unsigned long long wallet_get_pending_incoming_balance(struct TariWallet *wallet
 // Gets the outgoing balance from a TariWallet
 unsigned long long wallet_get_pending_outgoing_balance(struct TariWallet *wallet,int* error_out);
 
+// Get a fee estimate from a TariWallet for a given amount
+unsigned long long wallet_get_fee_estimate(struct TariWallet *wallet, unsigned long long amount, unsigned long long fee_per_gram, unsigned long long num_kernels, unsigned long long num_outputs, int* error_out);
```

Comment:
@delta1 When this merges, this method still needs to be added to the wrappers for iOS and Android. Just mentioning this here.

Reply:
thanks @StriderDM agreed